### DISCUSSION TITLE
Overlays Meeting (2024-07-30)

### DISCUSSION THREAD

**Discussion by lornajane** [OAI/Overlay-Specification#42] [Meeting Notes]
This meeting falls under the OpenAPI code of conduct. Everyone is welcome to join, we meet in alternate weeks.

Add any topics you'd like to discuss as comments on this discussion.

**Comment by lornajane** [comment]
We're working towards releasing the current state of the specification as 1.0.0, more or less without modification.

One thing we should do is reference the version of JSON path that we're using.

Anything else?

**Reply by lornajane** [reply]
I captured the question about the JSONPath version as #49

**Comment by handrews** [comment]
Comment in this week's meeting:  Need to work with a more abstract representation so that the exact file structure and reference arrangement does not get in the way.
